backends/arm/arm_backend.py

```diff
@@ -138,13 +156,28 @@ def preprocess(  # noqa: C901
 
                 # Visiting each Node
                 if node.target.__name__ in node_visitors:
-                    node_visitors[node.target.__name__].define_node(
-                        node, tosa_graph, inputs, output, is_quant_node(node)
-                    )
+                    if node.target.__name__ in [
+                        "aten.add.Tensor",
+                        "aten._native_batch_norm_legit_no_training.default",
```

Comment:
Other ops can handle !nhwc?

Reply:
yes, for each test case right now, we're annotating them either to do `permute to nhwc` or not by default

Reply:
But for a large network how is this handled when you have let's say
`input -> add* -> sub -> output`? Also why conv is not here?

Reply:
> But for a large network how is this handled when you have let's say
using the annotations below. 

> Also for conv is it always set?
yes 

I think this one can also be resolved given the annotation stuff is shown.